content/en/docs/pipelines/building-pipelines/overview.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Overview of Pipeline Development"
+description = "Getting started with pipeline development"
+weight = 10          
++++
+
+Kubeflow Pipelines lets you deploy and run portable, scalable machine learning
+(ML) workflows based on Docker containers. You can orchestrate your ML workflow
+as a pipeline using the Kubeflow Pipelines SDK or TensorFlow Extended (TFX).
+
+This document describes how to choose the best option for building your
+pipeline, and provides resources for getting started.
+
+## Selecting a pipelines SDK
+
+At a high level, the Kubeflow Pipelines SDK and TFX take a similar approaches
+to defining a pipeline. In both cases:
+
+*  A pipeline is a set of components and input parameters. 
+*  Component outputs are called artifacts. Components can accept either
+   pipeline parameters or the artifacts of other components as inputs.
+*  The execution order of pipeline components is a directed acyclic graph of
+   the pipeline's component artifact dependencies and any task-based
+   dependencies that were defined between components.
+*  The custom logic in a component is defined in a Python function or in a
+   container image.
+
+Use the Kubeflow Pipelines SDK, if:
+
+*  You rely on recursion, loops, graph components (pipelines that are executed
+   as a component of another pipeline), and conditional logic in the definition
+   of your pipeline's graph. 
+*  Your pipeline needs to manipulate Kubernetes resources, or you need to
+   specify resource dependencies at a component level. For example, your
+   training component may require GPUs and more memory than other steps in
+   your pipeline.
+*  Your model or pipeline is not a good match for TFX pipelines.
```

Comment:
not sure if it's possible to elaborate, but when would it not be a good match?

Reply:
I switched the order of the TFX and KFP sections, to help users understand if they are a good match for TFX before they get to that line.